OpenSim/Common/Reporter.h

```diff
@@ -228,7 +234,14 @@ OpenSim_DECLARE_CONCRETE_OBJECT_T(TableReporter_, InputT, Reporter<InputT>);
         for (auto idx = 0u; idx < input.getNumConnectees(); ++idx) {
             labels.push_back( input.getLabel(idx) );
         }
-        const_cast<Self*>(this)->_outputTable.setColumnLabels(labels);
+        if (!labels.empty()) {
+            const_cast<Self*>(this)->_outputTable.setColumnLabels(labels);
+        } else {
+            std::cout << "Warning: No outputs were connected to '"
+                      << this->getName() << "' of type "
+                      << getConcreteClassName()
+                      << "; did you intend to report any outputs?" << std::endl;
```

Comment:
The last piece of the message doesn't seem to add anything. Perhaps tell the user "You can connect outputs by calling `addToReport()`"?